tests/Builder/SourceFileRendererTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Builder;
+
+use DateTimeImmutable;
+use Doctrine\Website\Builder\SourceFile;
+use Doctrine\Website\Builder\SourceFileParameters;
+use Doctrine\Website\Builder\SourceFileRenderer;
+use Doctrine\Website\Controller\ControllerExecutor;
+use Doctrine\Website\Site;
+use Doctrine\Website\Twig\TwigRenderer;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class SourceFileRendererTest extends TestCase
+{
+    /** @var ControllerExecutor|MockObject */
+    private $controllerExecutor;
+
+    /** @var TwigRenderer|MockObject */
+    private $twigRenderer;
+
+    /** @var Site|MockObject */
+    private $site;
+
+    /** @var SourceFileRenderer */
+    private $sourceFileRenderer;
+
+    public function testRenderWithContentBlock() : void
+    {
+        $date                 = new DateTimeImmutable('2018-09-01');
+        $sourceFile           = $this->createMock(SourceFile::class);
+        $sourceFileParameters = new SourceFileParameters(['controller' => 'TestController']);
+        $controllerResult     = ['test' => true];
+        $contents             = 'Test';
```

Comment:
and with `$content`, this name would fit to the one of the test method. It's easier to read.

Reply:
@SenseException I don't quite understand, can you elaborate?

Reply:
I had problems to understand at first what these tests are for. I read into the code and understood it after I looked into it. The naming of the test method and this variable I suggested here would have made its intention easier to understand (for me).

And because the naming in the test method is singular, I suggested to use singular here too. Otherwhise it would be similar like this code example:

``` php
public function findProduct()
{
    // ... code code code
    return $products;
}
```